src/primitives/gf1024.rs

```diff
@@ -0,0 +1,14 @@
+// Written by Clark Moody and the rust-bitcoin developers.
+// SPDX-License-Identifier: MIT
+
+//! GF1024 - Galois Field over 1024 elements.
```

Comment:
I'm unclear on the purpose of including this in this PR.

Reply:
Will drop, thanks.